dev/tools/create_api_docs.dart

```diff
@@ -23,6 +23,58 @@ import 'dartdoc_checker.dart';
 const String kDummyPackageName = 'Flutter';
 const String kPlatformIntegrationPackageName = 'platform_integration';
 
+class PlatformDocsSection {
+  const PlatformDocsSection({
+    required this.zipName,
+    required this.sectionName,
+    required this.checkFile,
+    required this.subdir,
+  });
+  final String zipName;
+  final String sectionName;
+  final String checkFile;
+  final String subdir;
+}
+
+const Map<String, PlatformDocsSection> kPlatformDocs = <String, PlatformDocsSection>{
+  'android': PlatformDocsSection(
+    zipName: 'android-javadoc.zip',
+    sectionName: 'Android',
+    checkFile: 'io/flutter/view/FlutterView.html',
+    subdir: 'javadoc',
+  ),
+  'ios': PlatformDocsSection(
+    zipName: 'ios-docs.zip',
+    sectionName: 'iOS',
+    checkFile: 'interface_flutter_view.html',
+    subdir: 'ios-embedder',
+  ),
+  'macos': PlatformDocsSection(
+    zipName: 'macos-docs.zip',
+    sectionName: 'macOS',
+    checkFile: 'interface_flutter_view.html',
+    subdir: 'macos-embedder',
+  ),
+  'linux': PlatformDocsSection(
+    zipName: 'linux-docs.zip',
+    sectionName: 'Linux',
+    checkFile: 'struct___fl_view.html',
```

Comment:
is this supposed to have 3 underscores in a row?

Reply:
Yep.  That's how Doxygen generates it.

Reply:
:+1: